core/iwasm/common/wasm_runtime_common.h

```diff
@@ -369,9 +369,13 @@ typedef struct WASIContext {
     char **argv_list;
     char *env_buf;
     char **env_list;
+    uint32_t exit_code;
```

Comment:
I don't see where we initialize `exit_code`; is there something like `memset(ctx, 0, sizeof(WASIContext))` in the startup code to set the value of `exit_code` to 0? Otherwise it'll contain garbage if the module doesn't call `proc_exit`.